pkg/storj/object.go

```diff
@@ -91,6 +93,9 @@ type Segment struct {
 	// Encryption
 	EncryptedKeyNonce Nonce
 	EncryptedKey      EncryptedPrivateKey
+	// Pieces building info
+	Needed int32
+	Online int32
```

Comment:
These aren't properties of the segment, this is derived information and shouldn't be part of this.

Reply:
removed